src/App.css

```
@@ -0,0 +1,15 @@
+.Screen {
+  width: 360px;
+  height: 600px;
+  background: #fff;
+
+  display: flex;
+  flex-direction: column;
+
+  border-radius: 20px;
+  box-shadow: 0 0 25px rgba(105, 85, 85, 0.5);
+}
+
+.Container {
+  margin: 0% 4.5% 0% 4.5%;
```

`margin: 상하 좌우`의 형식으로 간단하게 쓸 수 있습니다

```suggestion
  margin: 0% 4.5%;
```

---

코드 정리할때 줄였어야 되는데 미처 확인하지 못했네요 😅